### DISCUSSION TITLE
vertical red lines

### DISCUSSION THREAD

**Discussion by zigius** [ghostty-org/ghostty#4079] [Q&A] [answered]
anyone seeing vertical red lines on the ghostty terminal?

**Comment by sitsofe** [comment, marked as answer]
> anyone seeing vertical red lines on the ghostty terminal?

Fullscreen, Intel Mac laptop? Similar to https://github.com/ghostty-org/ghostty/discussions/3352 ?

**Reply by zigius** [reply]
Yes, exactly

**Reply by zigius** [reply]
I'll close this thread
